Makefile

```diff
@@ -0,0 +1,121 @@
+ANSIBLE_DATA_DIR=${PKG_DATA_DIR}/roles
+SYMLINK_ROLES=true
+
+#
+# CUSTOMIZATION-END
+#
+
+include version.mak
+RPM_VERSION=$(VERSION)
+PACKAGE_VERSION=$(VERSION)$(if $(MILESTONE),_$(MILESTONE))
+DISPLAY_VERSION=$(PACKAGE_VERSION)
+
+TARBALL=$(PACKAGE_NAME)-$(PACKAGE_VERSION).tar.gz
+
+.SUFFIXES:
+.SUFFIXES: .in
+
+.in:
+	sed \
+	-e "s|@DATAROOT_DIR@|$(DATAROOT_DIR)|g" \
+	-e "s|@PKG_DATA_DIR=@|$(PKG_DATA_DIR=)|g" \
+	-e "s|@RPM_VERSION@|$(RPM_VERSION)|g" \
+	-e "s|@RPM_RELEASE@|$(RPM_RELEASE)|g" \
+	-e "s|@PACKAGE_NAME@|$(PACKAGE_NAME)|g" \
+	-e "s|@PACKAGE_VERSION@|$(PACKAGE_VERSION)|g" \
+	-e "s|@DISPLAY_VERSION@|$(DISPLAY_VERSION)|g" \
+	-e "s|@VERSION_MAJOR@|$(VERSION_MAJOR)|g" \
+	-e "s|@VERSION_MINOR@|$(VERSION_MINOR)|g" \
+	-e "s|@VERSION_PATCH_LEVEL@|$(VERSION_PATCH_LEVEL)|g" \
+	$< > $@
+
+
+.PHONY: ovirt-ansible-roles.spec.in
+
+dist:	ovirt-ansible-roles.spec
+	git ls-files | tar --files-from /proc/self/fd/0 -czf "$(TARBALL)" ovirt-ansible-roles.spec
+	@echo
+	@echo For distro specific packaging refer to http://www.ovirt.org/Build_Binary_Package
+	@echo
+
+copy-recursive:
+	( cd "$(SOURCEDIR)" && find . -type d -printf '%P\n' ) | while read d; do \
+		install -d -m 755 "$(TARGETDIR)/$${d}"; \
+	done
+	( \
+		cd "$(SOURCEDIR)" && find . -type f -printf '%P\n' | \
+		while read f; do echo "$${f}"; done \
+	) | while read f; do \
+		src="$(SOURCEDIR)/$${f}"; \
+		dst="$(TARGETDIR)/$${f}"; \
+		[ -x "$${src}" ] && MASK=0755 || MASK=0644; \
+		install -T -m "$${MASK}" "$${src}" "$${dst}"; \
+	done
+
+copy-doc:
+	( cd "$(SOURCEDIR)" && find . -maxdepth 1 -type d -printf '%P\n' ) | while read d; do \
+		install -d -m 755 "$(TARGETDIR)/$${d}"; \
+	done
+	( \
+	  cd "$(SOURCEDIR)" && find . -maxdepth 2 -name README.md -type f -printf '%P\n' | \
+          while read f; do echo "$${f}"; done \
+        ) | while read f; do \
+		src="$(SOURCEDIR)/$${f}"; \
+		dst="$(TARGETDIR)/$${f}"; \
+		[ -x "$${src}" ] && MASK=0755 || MASK=0644; \
+		install -T -m "$${MASK}" "$${src}" "$${dst}"; \
+	  done
+
+copy-examples:
+	( cd "$(SOURCEDIR)" && find . -maxdepth 1 -type d -printf '%P\n' ) | while read d; do \
+		install -d -m 755 "$(TARGETDIR)/$${d}"; \
+	done
+	( \
+	  cd "$(SOURCEDIR)" && find . maxdepth 1 -name '*yml' -type f -printf '%P\n' | \
+          while read f; do echo "$${f}"; done \
+        ) | while read f; do \
+		src="$(SOURCEDIR)/$${f}"; \
+		dst="$(TARGETDIR)/$${f}"; \
+		[ -x "$${src}" ] && MASK=0755 || MASK=0644; \
+		install -T -m "$${MASK}" "$${src}" "$${dst}"; \
+	  done
+
+create-license:
+	cp -f LICENSE "${DESTDIR}$(PKG_DOC_DIR)/"
+
+create-readme:
+	cp -f README.md "${DESTDIR}$(PKG_DOC_DIR)/"
+
+install: $(NULL)
+	$(MAKE) copy-recursive SOURCEDIR=roles TARGETDIR="${DESTDIR}$(ANSIBLE_DATA_DIR)"
+	$(MAKE) copy-doc SOURCEDIR=roles TARGETDIR="${DESTDIR}$(ANSIBLE_DOC_DIR)"
+	$(MAKE) copy-examples SOURCEDIR=examples TARGETDIR="${DESTDIR}$(PKG_DOC_DIR)/examples"
+	$(MAKE) create-license
+	$(MAKE) create-readme
```

Comment:
The reason we use Makefile instead of bash, is because in 4.2 we install some playbooks into the dev-env, and we want to be consistent with other projects which uses it as well.